File: POOPlatzi/PYTHON/uberBlack.py
from car import Car

class UberBlack(Car):
    
    tipoCarroAceptado=[]
    tipoCojineriaAcepatada=[]

    def __init__(self,licencia,conductor,tipoCarroAceptado,tipoCojineriaAceptada):

        super.__init__(licencia,conductor)
        self.tipoCarroAceptado=tipoCarroAceptado
        self.tipoCojineriaAcepada=tipoCojineriaAceptada

File: POOPlatzi/PHP/uberBlack.php
<?php

require_once('Car.php');
class  uberBlack extends Car{

public $tipoCarroAceptado;
public $tipoCojineriaAceptada;


public function __construct($licencia,$conductor,$tipoCarroAceptado,$tipoCojineriaAceptada){

parent::__construct($licencia,$conductor);

$this->tipoCarroAceptado=$tipoCarroAceptado;
$this->tipoCojineriaAceptada=$tipoCojineriaAceptada;




}


}
?>